dev/com.ibm.ws.messaging.runtime/src/com/ibm/ws/sib/admin/internal/JsLocalizer.java

```diff
@@ -229,14 +229,14 @@ private void addLpToMaps(LWMConfig lp) {
         lpMap.put(lpIdentifier, new LocalizationEntry(ld));
         String destName = lpIdentifier.substring(0, lpIdentifier.indexOf("@"));
 
-        // Get the MasterEntry object for this LPP. If it doesn't exist create one.
+        // Get the MainEntry object for this LPP. If it doesn't exist create one.
 
-        MasterEntry masterEntry = masterMap.get(destName);
+        MainEntry MainEntry = mainMap.get(destName);
```

Comment:
variable name should be `mainEntry` instead of `MainEntry`

Reply:
fixed!